### PR TITLE
Fix's the bug with the corgi dogs, makes mice butcherable.

### PR DESCRIPTION
**Pull request by dragoslava**
- rscadd: "Add mice to the butcher list"
  - bugfix: "Fixed the infinite corgi glitch by modifying the recipie to require corgi meat

### CODE REVIEW

**Comment by Kaedwuff** [comment]
"You have reached the epitome of poor, eating the poor mice of the station."

This sentence is really structured very awkwardly, I would recommend not repeating the word 'poor' twice in the same sentence.  For instance, "You have reached the epitome of poor, resorting to eating vermin."

Also, why do mice have poison in them?  Mice are not toxic or venomous.  You should not be inserting toxins into their meat.

File: code/game/machinery/kitchen/microwave.dm [hunk 1]
```diff
@@ -235,7 +235,7 @@
 *   Microwave Menu Handling/Cooking
 ************************************/
 
-/obj/machinery/microwave/proc/cook()
+/obj/machinery/microwave/proc/cook(var/mob/living/user)
```
Comment (Alberyk) [inline code/game/machinery/kitchen/microwave.dm]: Why is this needed?

Comment (dragoslava) [reply code/game/machinery/kitchen/microwave.dm]: Ignore, was testing around with a function and commited it. However after talking with skull, he apparently wants to stick with butchering the animal first as microwave code sucks ass in general.

**Comment by skull132** [comment]
I disagree, @Alberyk . All of the other animals are already processed by flaying, so why should the holder capable ones be special? It also makes more sense to flay and otherwise process them before shoving them into the microwave.

File: code/modules/mob/living/simple_animal/friendly/mouse.dm [hunk 2]
```diff
@@ -29,12 +29,12 @@
...
 	maxHealth = 5
 	health = 5
-	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/meat/mice
 	response_help  = "pets"
 	response_disarm = "gently pushes aside"
 	response_harm   = "stomps on"
 	density = 0
-	meat_amount = 1
+	meat_amount = 2
```
Comment (skull132) [inline code/modules/mob/living/simple_animal/friendly/mouse.dm]: I'd leave this at 1 t b h. Mice and small.

File: html/changelogs/miceandian.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Add mice to the butcher list"
```
Comment (skull132) [inline html/changelogs/miceandian.yml]: Periods missing from the changelog.

File: code/modules/reagents/reagent_containers/food/snacks/meat.dm [hunk 4]
```diff
@@ -91,3 +91,15 @@
...
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice
+	name = "mice meat"
+	icon_state = "chickenbreast"
+	desc = "You have reached the epitome of poor, eating the poor mice of the station."
+
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice/Initialize()
+	. = ..()
+	reagents.add_reagent("protein", 5)
+	reagents.add_reagent("toxin", 2)
```
Comment (LordFowl) [inline code/modules/reagents/reagent_containers/food/snacks/meat.dm]: Why exactly are mice poisonous? Considering historically speaking people ate mice without ill effect, I don't really see the intent here. If you meant to make them less nutritous, then you should use the bad nutrition that junk food has, although even that is questionable.

Comment (Kaedwuff) [reply code/modules/reagents/reagent_containers/food/snacks/meat.dm]: I said that earlier and they ignored me then. Maybe they'll listen this time when you say it.

File: code/modules/reagents/reagent_containers/food/snacks/meat.dm [hunk 5]
```diff
@@ -91,3 +91,15 @@
 	name = "snow strider meat"
 	desc = "A slab of nav'twir's meat, an animal native from Adhomai."
 	icon_state = "adhomai_meat"
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice
+	name = "mice meat"
+	icon_state = "chickenbreast"
+	desc = "You have reached the epitome of poor, eating the poor mice of the station."
```
Comment (LordFowl) [inline code/modules/reagents/reagent_containers/food/snacks/meat.dm]: The wording here is rather awkward and redundant; I suggest "You have reached the epitome of poorness: eating the station's vermin"

File: html/changelogs/miceandian.yml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+  - bugfix: "Fixed the infinite corgi glitch by modifying the recipie."
```
Comment (LordFowl) [inline html/changelogs/miceandian.yml]: Misspelled recipe.

File: html/changelogs/miceandian.yml (hunk 3 above)
Comment (LordFowl) [inline html/changelogs/miceandian.yml]: Should be "adds" not "add".

File: code/modules/reagents/reagent_containers/food/snacks/meat.dm (hunk 4 above)
Comment (dragoslava) [reply code/modules/reagents/reagent_containers/food/snacks/meat.dm]: Mice in a survival situation are abit more safer to eat then IE, a factory worker stabbing a mouse and cooking ti and eating it. The difference is the station is a industrial enviroment and mice as there nature crawl arouind and are exposed to harmful enviroments and diseases.

Comment (Kaedwuff) [reply code/modules/reagents/reagent_containers/food/snacks/meat.dm]: If you want to make the mice disease to figure out a way to give him a disease that can be transmitted through meat. Don't just arbitrarily make their meat toxic. This is a really clumsy implementation of what might be a good idea and needs Improvement if you really want to do it.

File: code/datums/diseases/blackcold.dm [hunk 7]
```diff
@@ -0,0 +1,67 @@
+/datum/disease/blackdeath
+	name = "The Black Death"
+	max_stages = 3
+	spread = "Airborne"
+	cure = "Spaceacillin"
+	cure_id = "spaceacillin"
+	cure_chance = 10
+	agent = "Bl3ck V3tosi"
```
Comment (MarinaGryphon) [inline code/datums/diseases/blackcold.dm]: What is this?

File: code/datums/diseases/blackcold.dm [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+					affected_mob.adjustHalLoss(20)
+					affected_mob.say( pick( list("AAAARRRGGHGHHHH", "...ffff.a.aahhh", "*groans*", "euaaycch!") ) )
+			if(prob(1))
+				affected_mob << "<span class='warning'>Your stomach hurts.</span>"
+				if(prob(20))
+					affected_mob.adjustToxLoss(1)
+					affected_mob.updatehealth()
+					affected_mob.adjustHalLoss(20)
+			if(prob(5))
+				affected_mob << "<span class='warning'>You slowly fall asleep sleep.</span>"
```
Comment (MarinaGryphon) [inline code/datums/diseases/blackcold.dm]: Extra `sleep` here.

File: code/datums/diseases/blackcold.dm [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+			if(prob(1))
+				affected_mob.emote("sneeze")
+			if(prob(1))
+				affected_mob.emote("cough")
+			if(prob(1))
+				affected_mob << "<span class='warning'>Your muscles ache.</span>"
+				if(prob(20))
+					affected_mob.take_organ_damage(1)
+					affected_mob.adjustHalLoss(20)
+					affected_mob.say( pick( list("AAAARRRGGHGHHHH", "...ffff.a.aahhh", "*groans*", "euaaycch!") ) )
```
Comment (MarinaGryphon) [inline code/datums/diseases/blackcold.dm]: They shouldn't be saying `*groans*`. Either make it have a chance to do an emote, or simply remove that one.
Also, proper capitalisation.

File: code/modules/reagents/reagent_containers/food/snacks/meat.dm [hunk 10]
```diff
@@ -91,3 +91,24 @@
...
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice
+	name = "mice meat"
+	icon_state = "chickenbreast"
+	desc = "You have reached the epitome of poorness: eating the station's vermin."
+
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice/Initialize()
+	. = ..()
+	reagents.add_reagent("protein", 5)
+	reagents.add_reagent("rattoxin", 1)
```
Comment (LordFowl) [inline code/modules/reagents/reagent_containers/food/snacks/meat.dm]: Giving people the bubonic plague is not better, In fact, I should think it's worse because now people can turn mice into bioweapons.

File: code/datums/diseases/blackcold.dm (hunk 8 above)
Comment (MarinaGryphon) [reply code/datums/diseases/blackcold.dm]: Additionally, this should use `to_chat(affected_mob, "text")` and `span("warning", "text")` instead.

File: code/datums/diseases/blackcold.dm [hunk 11]
```diff
@@ -0,0 +1,67 @@
...
+				affected_mob.vomit()
+
+			if(prob(10))
+				affected_mob << "<span class='warning'>You break into a deep sweat, screaming in agony.</span>"
+				affected_mob.vomit()
+				affected_mob.dizziness += 20
+				affected_mob.confused += 20
+				affected_mob.Jitter(20)
+				affected_mob.adjustHalLoss(20)
+				affected_mob.say( pick( list("AAAARRRGGHGHHHH", "...ffff.a.aahhh", "*groans*", "euaaycch!") ) )
```
Comment (MarinaGryphon) [inline code/datums/diseases/blackcold.dm]: Remove `*groans*` or rework it to be an emote. Also, capitalisation of `euaaycch!`.

File: code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm [hunk 12]
```diff
@@ -709,6 +709,21 @@
...
 
+
+
+/datum/reagent/rattoxin
+	name = "Toxins"
+	id = "rattoxin"
+	description = "Toxins, yuck!."
+	reagent_state = LIQUID
+	color = "#535E66"
+	taste_description = "eugh!"
```
Comment (MarinaGryphon) [inline code/modules/reagents/Chemistry-Reagents/Chemistry-Reagents-Toxins.dm]: This should probably be an adjective/noun.

File: code/modules/reagents/reagent_containers/food/snacks/meat.dm [hunk 13]
```diff
@@ -91,3 +91,24 @@
 	name = "snow strider meat"
 	desc = "A slab of nav'twir's meat, an animal native from Adhomai."
 	icon_state = "adhomai_meat"
+
+/obj/item/weapon/reagent_containers/food/snacks/meat/mice
+	name = "mice meat"
+	icon_state = "chickenbreast"
+	desc = "You have reached the epitome of poorness: eating the station's vermin."
```
Comment (MarinaGryphon) [inline code/modules/reagents/reagent_containers/food/snacks/meat.dm]: `poverty` instead of `poorness`, as `poorness` isn't a word.

File: html/changelogs/miceandian.yml [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes: 
+  - rscadd: "Adds mice and nymphs to the butcher list."
+  - rscadd: "Eating mice now has side effects."
```
Comment (MarinaGryphon) [inline html/changelogs/miceandian.yml]: Probably needs to mention that it may be lethal.

File: code/datums/diseases/blackcold.dm [hunk 15]
```diff
@@ -0,0 +1,67 @@
...
+	name = "The Black Death"
+	max_stages = 3
+	spread = "Airborne"
+	cure = "Spaceacillin"
+	cure_id = "spaceacillin"
+	cure_chance = 10
+	agent = "Bl3ck V3tosi"
+	affected_species = list("Human", "Monkey")
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will feel a insane fever and slowly succumb into a coma. Caused by eating mice" //Mice shaming is a good thing --Drago
```
Comment (MarinaGryphon) [inline code/datums/diseases/blackcold.dm]: Severe fever, not insane. One slips into a coma and doesn't succumb to it. Needs a period at the end.

File: aurorastation.dme [hunk 16]
```diff
@@ -249,6 +249,7 @@
 #include "code\datums\discord\webhook.dm"
 #include "code\datums\diseases\appendicitis.dm"
 #include "code\datums\diseases\beesease.dm"
+#include "code\datums\diseases\blackcold.dm"
```
Comment (MarinaGryphon) [inline aurorastation.dme]: `blackdeath.dm` instead, please.

File: code/modules/reagents/reagent_containers/food/snacks/meat.dm (hunk 13 above)
Comment (LordFowl) [reply code/modules/reagents/reagent_containers/food/snacks/meat.dm]: Poorness is a word.

**Comment by skull132** [comment]
Just. Remove the plague thing. And we can finally merge this one.

**Comment by dragoslava** [comment]
fixed

**Comment by dragoslava** [comment]
!review
